### PR TITLE
feat(apps/gql): add pano post resolvers

### LINKED ISSUE #512: feat(gql): Create GQL resolvers & loaders for fetching a post

### LINKED ISSUE #515: feat(gql): Create GQL resolvers & loaders for fetching single post's owner

### PR DESCRIPTION
**Pull request by usirin**
Part of #510 

Closes #512 
Closes #515 

cc @FurkanGM @durmusrasit @Ketcap

### CODE REVIEW

File: db/prisma/seed.ts [hunk 1]
```diff
@@ -139,8 +138,11 @@ async function seedAll(users: User[], posts: Post[]) {
       select: { id: true },
     });
 
+    console.log(">> post", post);
```
Comment (cansirin) [inline db/prisma/seed.ts]:
```suggestion
```

File: db/prisma/seed.ts [hunk 2]
```diff
@@ -139,8 +138,11 @@ async function seedAll(users: User[], posts: Post[]) {
...
+    console.log(">> prismaPost", prismaPost);
```
Comment (cansirin) [inline db/prisma/seed.ts]:
```suggestion
```

File: db/prisma/seed.ts [hunk 3]
```diff
@@ -139,8 +138,11 @@ async function seedAll(users: User[], posts: Post[]) {
...
+
     if (prismaPost) {
-      return;
+      continue;
```
Comment (cansirin) [inline db/prisma/seed.ts:143]: maybe null check and return? 👀

Comment (usirin) [reply db/prisma/seed.ts:143]: naah, this is a for loop, it was a return before and it was breaking the loop AND the function lol :D

Comment (cansirin) [reply db/prisma/seed.ts:143]: haha, sounds fair :D

**Review by cansirin — APPROVED** [review]
perfecto 💯

**Pull request merged**